zen-fribidi.sh

```diff
@@ -1,34 +1,34 @@
-#!/bin/bash
-
-# Copyright (C) 2014 - Romain Bouqueau
-# This program is free software; you can redistribute it and/or
-# modify it under the terms of the GNU General Public License
-# as published by the Free Software Foundation; either version 3
-# of the License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
-
-
-function build_fribidi {
-  host=$1
-  pushDir $WORK/src  
-
-  lazy_download "fribidi.tar.xz" "http://fribidi.org/download/fribidi-0.19.6.tar.bz2"
-  lazy_extract "fribidi.tar.xz"
-
-  mkgit "fribidi"
-
-  autoconf_build $host "fribidi" --disable-shared

```

Comment:
When building shared, fribidi won't build in MSys2:
```
  CC       fribidi-char-sets.lo
  CC       fribidi-char-sets-cap-rtl.lo
  CC       fribidi-char-sets-cp1256.lo
  CC       fribidi-char-sets-cp1255.lo
  CC       fribidi-char-sets-iso8859-6.lo
  CC       fribidi-char-sets-utf8.lo
  CC       fribidi-char-sets-iso8859-8.lo
  CCLD     libfribidi-char-sets.la
make[2]: Leaving directory '/c/Works/zenbuild/zenbuild_github_badr/20150205_vlc/src/fribidi/build/x86_64-w64-mingw32/charset'
Making all in lib
make[2]: Entering directory '/c/Works/zenbuild/zenbuild_github_badr/20150205_vlc/src/fribidi/build/x86_64-w64-mingw32/lib'
make[2]: AVERTISSEMENT : -jN forcé dans un submake : désactivation du mode serveur de tâches.
make -j8  all-am
make[3]: Entering directory '/c/Works/zenbuild/zenbuild_github_badr/20150205_vlc/src/fribidi/build/x86_64-w64-mingw32/lib'
make[3]: AVERTISSEMENT : -jN forcé dans un submake : désactivation du mode serveur de tâches.
  CC       fribidi.lo
  CC       fribidi-arabic.lo
  CC       fribidi-bidi.lo
  CC       fribidi-bidi-types.lo
  CC       fribidi-deprecated.lo
  CC       fribidi-joining.lo
  CC       fribidi-joining-types.lo
  CC       fribidi-mem.lo
  CC       fribidi-mirroring.lo
  CC       fribidi-run.lo
  CC       fribidi-shape.lo
  CCLD     libfribidi.la
make[3]: Leaving directory '/c/Works/zenbuild/zenbuild_github_badr/20150205_vlc/src/fribidi/build/x86_64-w64-mingw32/lib'
make[2]: Leaving directory '/c/Works/zenbuild/zenbuild_github_badr/20150205_vlc/src/fribidi/build/x86_64-w64-mingw32/lib'
Making all in bin
make[2]: Entering directory '/c/Works/zenbuild/zenbuild_github_badr/20150205_vlc/src/fribidi/build/x86_64-w64-mingw32/bin'
make[2]: AVERTISSEMENT : -jN forcé dans un submake : désactivation du mode serveur de tâches.
  CC       fribidi-main.o
  CC       getopt.o
  CC       getopt1.o
  CC       fribidi-benchmark.o
  CC       fribidi-bidi-types.o
  CC       fribidi-caprtl2utf8.o
  CCLD     fribidi-bidi-types.exe
  CCLD     fribidi-caprtl2utf8.exe
  CCLD     fribidi.exe
  CCLD     fribidi-benchmark.exe
fribidi-caprtl2utf8.o:fribidi-caprtl2utf8.c:(.text.startup+0x30): undefined reference to `fribidi_parse_charset'
fribidi-caprtl2utf8.o:fribidi-caprtl2utf8.c:(.text.startup+0x3f): undefined reference to `fribidi_parse_charset'
fribidi-caprtl2utf8.o:fribidi-caprtl2utf8.c:(.text.startup+0xe8): undefined reference to `fribidi_charset_to_unicode'
fribidi-caprtl2utf8.o:fribidi-caprtl2utf8.c:(.text.startup+0x110): undefined reference to `fribidi_unicode_to_charset'
E:/Users/rbouqueau/Documents/Works/software/msys2/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.1/../../../../x86_64-w64-mingw32/bin/ld.exe: fribidi-caprtl2utf8.o: bad reloc address 0x0 in section `.pdata.startup'
E:/Users/rbouqueau/Documents/Works/software/msys2/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.1/../../../../x86_64-w64-mingw32/bin/ld.exe: final link failed: Invalid operation
collect2.exe: error: ld returned 1 exit status
Makefile:457: recipe for target 'fribidi-caprtl2utf8.exe' failed
make[2]: *** [fribidi-caprtl2utf8.exe] Error 1
```

May be related to the fridibi version.